include/jwt-cpp/jwt.h

```diff
@@ -586,6 +579,28 @@ namespace jwt {
 			return {ptr, static_cast<size_t>(len)};
 		}
 
+		/**
+		 * \brief Convert the certificate provided as base64 DER to PEM.
+		 *
+		 * This is useful when using with JWKs as x5c claim is encoded as base64 DER. More info
+		 * (here)[https://tools.ietf.org/html/rfc7517#section-4.7]
+		 *
+		 * \tparam Decode is callabled, taking a string_type and returns a string_type.
+		 * It should ensure the padding of the input and then base64 decode and return
+		 * the results.
+		 *
+		 * \param cert_base64_der_str 	String containing the certificate encoded as base64 DER
+		 * \param decode 				The function to decode the cert
+		 * \param ec					error_code for error_detection (gets cleared if no error occures)
+		 */
+		template<typename Decode>
+		std::string convert_base64_der_to_pem(const std::string& cert_base64_der_str, Decode decode,
+											  std::error_code& ec) {
+			ec.clear();
+			const auto decodedStr = decode(cert_base64_der_str);
```

Comment:
I am on a holy war to free the world from the burden of camelCase and as such the entire codebase uses snake_case (or no spacing). Both `decoded_str`, `decodedstr` and `decoded` are fine with me.

Reply:
Yeah to be honest not sure how that got there as I use `camel_case`.